packages/app-content-pages/next.config.js

```diff
@@ -12,9 +12,13 @@ setAliases({
   'styled-components': resolveHoisted('styled-components')
 })
 
+function commitID () {
+  return execSync('git rev-parse HEAD').toString('utf8').trim()
```

Comment:
not blocking, i assume the git dep is available in the docker image?

Reply:
I assume so, since we haven’t seen this error during builds. I think passing the commit from Jenkins will be more robust.